.github/workflows/test.yml

```diff
@@ -12,9 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:  # https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support
-        django-version: [50]
+        django-version: [50, 51]
         python-version: [3.10, 3.11, 3.12]
```

Comment:
Need 3.13 in here, I think...?

Reply:
Right, got blurred by the exclude list ..